config/schema.yaml

```diff
@@ -80,6 +80,11 @@ germline_resource_gnomad_vcf:
   type: 'Path'
   required: false
   allow_empty: true
+panel_of_normals_vcf:
+  type: 'Path'
+  required: false
+  allow_empty: true
```

Comment:
A type of `Path` doesn't have an `allow_empty` property at the moment so I would suggest removing it to prevent confusion

Reply:
I removed this for `germline_resource_gnomad_vcf`, too